api/proxy/proxy.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package proxy;
+
+service Proxy {
+    rpc Backup (BackupRequest) returns (BackupReply) {}
+    rpc Restore (RestoreRequest) returns (RestoreReply) {}
+}
+
+message BackupRequest {
+    string cluster_identifier = 1;
+    google.protobuf.Timestamp backup_timestamp = 2;
+    bytes backup = 3;
+}
+
+message BackupReply {
+    string backup_url = 1;
+}
+
+message RestoreRequest {
+    string cluster_identifier = 1;
+    string backup_url = 2;
+}
+
+message RestoreReply {
```

Comment:
Can we name these something like `store` and `get` maybe? This ultimately is a very simple wrapper around PUT and GET operations.

Reply:
We use the verbs `upload` and `download` verbs in the permissions model. How about `Upload` and `Download` in that case. (So this spesific`message` on line 27 would be `DownloadReply`).

Reply:
upload and download works for me 😄

Reply:
Renamed.